extensions/azdata/src/common/childProcess.ts

```diff
@@ -51,16 +59,23 @@ export async function executeCommand(command: string, args?: string[], outputCha
  * @param args The additional args
  * @param outputChannel Channel used to display diagnostic information
  */
-export async function executeSudoCommand(command: string, outputChannel?: vscode.OutputChannel): Promise<string> {
-	return new Promise<string>((resolve, reject) => {
-		outputChannel?.appendLine(loc.executingCommand(`sudo ${command}`, []));
+export async function executeSudoCommand(command: string, outputChannel: vscode.OutputChannel): Promise<{ stdout: string, stderr: string }> {
```

Comment:
Create a type for this perhaps especially since it is used both here and in executeCommand